settings/common.py

```diff
@@ -284,7 +284,8 @@
 
 # To make usermame field read-only, customized serializer is defined.
 REST_AUTH_SERIALIZERS = {
-    "USER_DETAILS_SERIALIZER": "accounts.serializers.ProfileSerializer"
+    "USER_DETAILS_SERIALIZER": "accounts.serializers.ProfileSerializer",
+    "PASSWORD_RESET_SERIALIZER": "accounts.serializers.CustomPasswordResetSerializer"
```

Comment:
Change `accounts.serializers.CustomPasswordResetSerializer` to `accounts.serializers.PasswordResetSerializer`

Reply:
Done